pkg/proxy/ipvs/proxier.go

```diff
@@ -377,6 +377,9 @@ func NewProxier(ipt utiliptables.Interface,
 	}
 	if kernelVersion.LessThan(version.MustParseGeneric(connReuseMinSupportedKernelVersion)) {
 		klog.ErrorS(nil, fmt.Sprintf("can't set sysctl %s, kernel version must be at least %s", sysctlConnReuse, connReuseMinSupportedKernelVersion))
+	} else if kernelVersion.AtLeast(version.MustParseGeneric("5.9")) {
```

Comment:
can we assign a constant variable for "5.9" similar to connReuseMinSupportedKernelVersion and maybe a comment for why we are not setting it start in 5.9?

Reply:
It is a valuable fix for users with 5.9+ kernels. 
I suggest that we leave it a cleanup task later as today/tomorrow is the code freeze deadline?  I will open a cleanup task for this (and a new UT case to cover this is appreciated, I think. And a document to highlight things that is related to the kernel version is helpful. ). 
/lgtm

BTW, as @cofyc  asked in https://github.com/kubernetes/kubernetes/issues/93297#issuecomment-770373290, should we cherry-pick it to 1.21 and old releases?

Reply:
Constant added with a ref to the kernel commit